.prettierignore

```diff
@@ -16,6 +16,7 @@
 *.env
 *.sh
 *.json
+*.local
```

Comment:
Was this file accidentally changed and added to this PR? Or, is it something that is needed because `sites/sde/env.local` was added to the repo?